<launch>
    <arg name="visualization" default="true"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_planner" type="test_ics" name="terrain_planner" output="screen">
        <param name="visualize" value="$(arg visualization)"/>
        <param name="map_path" value="$(find terrain_planner)/../resources/sertig.tif"/>
        <param name="color_file_path" value="$(find terrain_planner)/../resources/sertig_color.tif"/>
        <param name="output_file_path" value="$(find terrain_planner)/../output/output.csv"/>
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_planner)/launch/config_ics.rviz" />
    </group>
</launch>
